config.yml

```diff
@@ -1,4 +1 @@
-API_ENDPOINT: api.figo.me
-FINGER_PRINTS:
-  - "79:B2:A2:93:00:85:3B:06:92:B1:B5:F2:24:79:48:58:3A:A5:22:0F:C5:CD:E9:49:9A:C8:45:1E:DB:E0:DA:50"  # valid until Jun 24 23:59:59 2018
-  - "CD:F3:D3:26:27:89:91:B9:CD:AE:4B:10:6C:96:81:B7:EB:B3:38:10:C4:72:37:6A:4D:9C:84:B7:B3:DC:D6:8D"  # valid until Jun 24 23:59:59 2019
+API_ENDPOINT: api.figo.me
```

Comment:
Please add a newline at the end of file. Otherwise the last byte sequence of the file is not considered [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) standard compliant line.